application/core/plugins/Authdb/Authdb.php

```diff
@@ -171,7 +178,7 @@ public function newUserSession()
             $this->setAuthFailure(self::ERROR_USERNAME_INVALID);
             return;
         }
-        if ($user->isExpired()) {
+        if ($user->isExpired() || (isset($user->user_status) && !$user->user_status)) {
```

Comment:
I think this check has to be moved to

application/core/LSUserIdentity.php
 71                 $authEvent = new PluginEvent('newUserSession', $this); // TODO: rename the plugin function authenticate()                                                                  

Basically, never fire up the event "newUserSession" if username belongs to a non-active user.

Reply:
Ah yeah, I did think about how to achieve that.

Reply:
But then we get the same issue with updates, or?

Reply:
> But then we get the same issue with updates, or?

Yes, but need to be fixed on the same way :)